templates/search.html

```diff
@@ -42,5 +44,52 @@ <h2><a href="/dataset/{{ result.source }}/{{ result.id }}">{{ result.dataset.tit
 
     </p>
 
+    <script>
```

Comment:
The JavaScript code is probably becoming large enough that it should be shipped as a separate asset using [`ServeDir`](https://docs.rs/tower-http/latest/tower_http/services/fs/struct.ServeDir.html) or by include it in the binary using [`include_str!`](https://doc.rust-lang.org/stable/std/macro.include_str.html).

Reply:
#86 adds the infrastructure to make use of `ServeDir` for arbitrary assets.